bears/general/InvalidLinkBear.py

```diff
@@ -33,14 +44,15 @@ def find_links_in_file(file):
             match = regex.search(line)
             if match:
                 link = match.group()
-                code = InvalidLinkBear.get_status_code_or_error(link)
+                code = self.get_status_code_checked(link)
                 yield line_number + 1, link, code
 
     def run(self, filename, file):
         '''
         Yields results for all invalid links in a file.
         '''
-        for line_number, link, code in InvalidLinkBear.find_links_in_file(file):
+        for line_number, link, code in (InvalidLinkBear.find_links_in_file(
+                                        self, file)):
```

Comment:
No in this PR, in a separate commit, since it is not associated with the current commit anyway.